pom.xml

```diff
@@ -327,6 +344,12 @@
             <artifactId>logback-core</artifactId>
             <version>${logback-version}</version>
          </dependency>
+         <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok-version}</version>
+            <scope>provided</scope>
```

Comment:
I think the scope needs to be removed here and only moved to where the dependency is used.